tests/test_features.py

```diff
@@ -691,15 +702,6 @@ def test_rasterize_invalid_fill_value(basic_geometry):
     assert out.dtype.kind != "f"
 
 
-def test_rasterize_fill_value_dtype_mismatch(basic_geometry):
-    """A fill value that doesn't match dtype should fail."""
-    with pytest.raises(ValueError):
-        rasterize(
-            [basic_geometry], out_shape=DEFAULT_SHAPE, fill=1000000,
-            default_value=2, dtype=np.uint8
-        )
```

Comment:
The ValueError raised previously had nothing to do with the fill value. The test was invalid.